pkg/simpleschema/transform.go

```diff
@@ -212,8 +215,39 @@ func (tf *transformer) applyMarkers(schema *extv1.JSONSchemaProps, markers []*Ma
 			if val, err := strconv.ParseFloat(marker.Value, 64); err == nil {
 				schema.Maximum = &val
 			}
+		case MarkerTypeEnum:
+			enumValues := strings.Split(marker.Value, ",")
+
+			var enumJSONValues []extv1.JSON
+
+			for _, val := range enumValues {
+				val = strings.TrimSpace(val)
+
+				if val == "" {
+					return fmt.Errorf("empty enum values are not allowed")
+				}
+
+				var rawValue []byte
+
+				switch schema.Type {
```

Comment:
nit: while not specific to Go, it's a common practice to group related statements more closely together with fewer blank lines between them. Typically you'll see blank lines only to separate logical "paragraphs" of code
```suggestion
			var enumJSONValues []extv1.JSON

			enumValues := strings.Split(marker.Value, ",")
			for _, val := range enumValues {
				val = strings.TrimSpace(val)
				if val == "" {
					return fmt.Errorf("empty enum values are not allowed")
				}
				
				var rawValue []byte
				switch schema.Type {
```

Reply:
Done with all suggestions 👍🏻